### PR TITLE
doc(ESS Kit): Inital commit

### PR DESCRIPTION
**Pull request by stephanbcbauer**
## Description

This PR adds the ESS Kit to our [tractus-x webpage](https://eclipse-tractusx.github.io/Kits). This PR is created for [birgitZF](https://github.com/birgitZF)

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [X] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [ ] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Review by birgitZF — COMMENTED** [review]
adjust table structure for chapter incident personas

**Review by birgitZF — COMMENTED** [review]
> @maximilianong , @birgitZF , @danielmiehle What I currently see as tasks:

- [x] Notice section with Copyright -> @birgitZF -> there is a need for additional section in the files, called Notice [e.g](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/blob/main/docs-kits/kits/Connector%20Kit/Adoption%20View/adoption-view.md#notice)
- [ ] New Banner for Kit -> @maximilianong 
- [ ] Adapt the table(s) -> @birgitZF -> Here I would need the original table to support you
- [ ] Check the used pictures -> are we allowed to use them?
- [ ] Naming of pictures -> "Catena-X" -> is this allowed?

**Comment by birgitZF** [comment]
chapter 1.2.3 ESS Incidents personas
[table1_chapter_ESS Incidents personas .docx](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/files/14370278/table1_chapter_ESS.Incidents.personas.docx)

chapter 1.2.4 Semantic models
[table2_chapter_Semantic models .docx](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/files/14370290/table2_chapter_Semantic.models.docx)

**Comment by SebastianBezold** [comment]
> added notice section with copyright open question 1: year = 2024, 2024? (like other kits who have 2023,2023?), open question 2: please check URL
> 
> i ajdusted the tables format (similar to pcf example now), check please. if not ok, I will send tables - original in pdf/docx

Hi @birgitZF,

I can try to answer that to the best of my knowledge :)
- If using a comma, we indicate a date range -> same year does not make sense, we should also tell the other KIT teams
- There have been updates to the copyright section in the [Eclipse Foundation Project Handbook](https://www.eclipse.org/projects/handbook/#ip-copyright-headers). Instead of providing a range, we also do have the option to just state the year of creation and do not provide addional ranges, which would need to be updated on each change in a new year. 
- We use EITHER ranges OR only the creation date. Not sure if we should generally align that
- I think we should use the repository URL as the source in the Notice section: https://github.com/eclipse-tractusx/eclipse-tractusx.github.io

**Comment by birgitZF** [comment]
@SebastianBezold thanks for your input, I have changed the range yyyy, yyyy

@stephanbcbauer can you check the format of the chapters with tables, please?

**Comment by stephanbcbauer** [comment]
> @SebastianBezold thanks for your input, I have changed the range yyyy, yyyy
> 
> @stephanbcbauer can you check the format of the chapters with tables, please?

@birgitZF sorry Haven't had time yet. Do we have a deadline?

File: docs-kits/kits/ESS Kit/images/CatenaX_ESS_KitBlockDiagram.png
Comment (maximilianong) [inline docs-kits/kits/ESS Kit/images/CatenaX_ESS_KitBlockDiagram.png:1]: are all those images self-created and only published here or were they als published in a standard first. Be aware of the legal notice: https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-08/#self-created-images

File: docs-kits/kits/ESS Kit/page_development-view.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+- SPDX-License-Identifier: CC-BY-4.0
+- SPDX-FileCopyrightText: 2022, 2024 ZF Friedrichshafen AG
+- SPDX-FileCopyrightText: 2022, 2024 BASF SE
+- SPDX-FileCopyrightText: 2022, 2024 Mercedes Benz Group
+- SPDX-FileCopyrightText: 2022, 2024 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+- SPDX-FileCopyrightText: 2022, 2024 SAP SE
+- SPDX-FileCopyrightText: 2022, 2024 Robert Bosch GmbH
+- SPDX-FileCopyrightText: 2022, 2024 Gris Group
+- SPDX-FileCopyrightText: 2022, 2024 Contributors to the Eclipse Foundation
+- Source URL: https://github.com/eclipse-tractusx/ESS-kit
```
Comment (maximilianong) [inline docs-kits/kits/ESS Kit/page_development-view.md:52]: Why is there 2022, 2024? This one got published 2024 correct?

File: docs-kits/kits/ESS Kit/images/CatenaX_ESS_KitSequenceView.png
Comment (maximilianong) [inline docs-kits/kits/ESS Kit/images/CatenaX_ESS_KitSequenceView.png:1]: This is not readable - when you zoom in it gets very unsharp. Please add this as .svg or even better add this as a mermaid diagram https://github.com/mermaid-js/mermaid

File: docs-kits/kits/ESS Kit/page_development-view.md
Comment (maximilianong) [inline docs-kits/kits/ESS Kit/page_development-view.md:1]: This not really a development view - this is more like high level architecture right? I think you should add this more like this - example: https://eclipse-tractusx.github.io/docs-kits/kits/Business%20Partner%20Kit/Documentation%20BPDM/bpdm_arc42

File: docs-kits/kits/ESS Kit/page_development-view.md (hunk 1 above)
Comment (birgitZF) [reply docs-kits/kits/ESS Kit/page_development-view.md:52]: yes, but we worked on them from 2022 until 2024.  
in the comment from Sebastian  (see above) : "..If using a comma, we indicate a date range -> same year does not make sense, we should also tell the other KIT teams.."

**Comment by stephanbcbauer** [comment]
@birgitZF, @SebastianBezold, @maximilianong, @danielmiehle what I did with my last commit
- move the images to a new static folder as discussed -> if it doesn't fit, let me know
- rename development view to documentation
- restructure tables -> isn't that nice, but maybe it's enough

@birgitZF please check the tables, cause it isnt that easy, had to restructure a little bit
image


@birgitZF instead of the png i used the SVG, but i think you have to invert the colours here?
image


@maximilianong I think we still need a KIT Banner/icon?

**Comment by birgitZF** [comment]
@stephanbcbauer table 1 is ok, table 2:  headline first table section = "ESS Incident Information" instead of "ESS incident Data Model". for me svg-file looks like this, may-be depending on browser settings?
image

**Comment by stephanbcbauer** [comment]
> @stephanbcbauer can you review the sequence diagram, I added white rect but did not manage to add complete white background. may-be it is ok like this? Alternatively we could use sequence-png - file?

@birgitZF I think it's not 100% perfect :)

image

**Comment by stephanbcbauer** [comment]
@birgitZF not sure why, but I think you also pushed a file accidentally -> we should delete it. Thought these files are on gitignore

image

**Comment by birgitZF** [comment]
I deleted file settings.json in folder .vscode, BUT no change for commit came up @stephanbcbauer do I need to do something else?

**Comment by stephanbcbauer** [comment]
> I deleted file settings.json in folder .vscode, BUT no change for commit came up @stephanbcbauer do I need to do something else?

We already solved it, correct?

**Comment by stephanbcbauer** [comment]
> > I deleted file settings.json in folder .vscode, BUT no change for commit came up @stephanbcbauer do I need to do something else?
> 
> We already solved it, correct?

@birgitZF can we please clean up the PR a little bitte? Check all the tasks which are already done. Resolve conversation if done. From my point of view only the KIT Picture/Logo/Banner is open?

**Comment by stephanbcbauer** [comment]
> the remaining open points are: @maximilianong -KIT picture / Logo/ Banner missing -Naming of pictures -> "Catena-X" -> is this allowed?
> 
> don't know how to "resolve conversation"

@birgitZF in some conversation, there is a button (if the point is solved, please click it :))
image

**Comment by maximilianong** [comment]
> @birgitZF , @danielmiehle , @maximilianong can I help here? Or is it done? @maximilianong Do we get a Banner/Icon for this KIT? Or should we just delete the placeholder?

I sent you something via MS Teams

**Comment by birgitZF** [comment]
@maximilianong I did not receive anything, can you send again or attach it here?

**Comment by maximilianong** [comment]
> @maximilianong I did not receive anything, can you send again or attach it here?

I sent it to @stephanbcbauer 

ESS
SustainabilityKIT

**Comment by stephanbcbauer** [comment]
Ah sorry didn't see it. Will add it with the next commit.

**Comment by stephanbcbauer** [comment]
@birgitZF, @maximilianong added all the icons and also added the KIT to the kits list -> it is shown on the landing page

image

image

File: docs-kits/kits/ESS Kit/images/CatenaX_ESS_KitBlockDiagram.png
Comment (stephanbcbauer) [reply docs-kits/kits/ESS Kit/images/CatenaX_ESS_KitBlockDiagram.png:1]: @birgitZF is this is OK? You created the pictures on your own. Correct? Please resolve the conversation.

**Review by birgitZF — APPROVED** [review]
pictures created by our ESS team, checkbox already checked

File: docs-kits/kits/ESS Kit/images/CatenaX_ESS_KitBlockDiagram.png
Comment (birgitZF) [reply docs-kits/kits/ESS Kit/images/CatenaX_ESS_KitBlockDiagram.png:1]: all images are created by ourselves (sorry, I do not see a "resolve button" anywhere)

**Review by maximilianong — APPROVED** [review]

**Review by danielmiehle — APPROVED** [review]

**Pull request merged**